layouts/applications/single.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html itemscope itemtype="http://schema.org/WebPage" lang="{{ .Site.Language.Lang }}" class="no-js">
+  <head>
+    <meta http-equiv="refresh" content="0; url='/applications/?id={{.Title | urlize}}'" />
+    {{ partial "head.html" . }}
```

Comment:
Because even if this page is just redirecting, but as this page will be there in google searches, so having `meta` and SEO tags might a good idea as if someone shares this page on any social media, we should be providing meta tags for that. These are my thoughts, what do you think?

Reply:
Sounds good